electron/main.ts

```diff
@@ -249,1649 +252,323 @@ const template: Electron.MenuItemConstructorOptions[] = [
- * @returns - the definition of list respective to list id
- */
-const fetchListDefinition = (event, listId) => {
-    log.info(`fetchListDefinition, listId: ${listId}`);
-    try {
-        const fetchedRows = db.prepare('SELECT * from lists where list_id = ? limit 1').get(listId) as any;
-
-        if (fetchedRows) {
-            event.returnValue = {
-                filterDefinition: fetchedRows.filter_definition,
-                columnDefinition: fetchedRows.column_definition,
-                datasource: fetchedRows.datasource,
-                listName: fetchedRows.list_name,
-                listHeader: fetchedRows.list_header,
-            };
-        } else {
-            event.returnValue = null;
-        }
-    } catch (err) {
-        log.info(`fethcListDefinition Error, listId: ${listId}`);
-    }
-};
-
-function getCurrentUser() {
-    const query = 'SELECT username from users LIMIT 1';
-    const fetchedRow = db.prepare(query).get() as any;
-    return fetchedRow.username;
-}
-
-const fetchFormListDefinition = (event, formId) => {
-    log.info(`fetchFormListDefinition, listId: ${formId}`);
-    try {
-        const userName = getCurrentUser();
-        const query = 'SELECT * from data where form_id = ? and submitted_by = ?';
-        const fetchedRows = db.prepare(query).all(formId, userName);
-
-        event.returnValue = { fetchedRows };
-    } catch (err) {
-        log.info(`fetchFormListDefinition, listId: ${formId}`);
-    }
-};
-
-const fetchFollowupFormData = (event, formId, detailsPk, pkValue, constraint) => {
-    log.info(`fetchFollowupFormData, formId: ${formId} ${detailsPk}, ${pkValue}, ${constraint}`);
-    try {
-        let query;
-        if (constraint == 'equal') {
-            query =
-                "SELECT data_id, submitted_by, submission_date, data from data where form_id = ? and json_extract(data, '$.?') = ?";
-        } else {
-            query =
-                "SELECT data_id, submitted_by, submission_date, data from data where form_id = ? and json_extract(data, '$.?') LIKE '%?%'";
-        }
-        const fetchedRows = db.prepare(query).all(formId, detailsPk, pkValue);
-
-        event.returnValue = { fetchedRows };
-    } catch (err) {
-        log.info(`fetchFollowupFormData, formId: ${formId}`);
-    }
-};
+const signIn = async (event, userData) => {
+    log.info(`Attempting electron-side signIn for ${userData.username}`);
+    log.debug(event);
 
-/** fetches the data based on query
- * @param {IpcMainEvent} event - the default ipc main event
- * @param {string} queryString- the query string
- * @returns - the returned dataset from the query
- */
-const fetchQueryData = (event, queryString) => {
-    log.info(`fetchQueryData, formId: ${queryString}`);
-    try {
-        const fetchedRows = db.prepare(queryString).all();
+    const SIGN_IN_ENDPOINT = `${BAHIS2_SERVER_URL}/bhmodule/app-user-verify/`;
+    const current_user = db.prepare('SELECT * from users2 limit 1').get();
```

Comment:
@ChasNelson1990 Why here table is `users2`

Reply:
Because it is still the original BAHIS 2 user table. I did refine it so it should be ready for BAHIS 3 auth but for now it's still the old system.